pipelines/smartseq2_single_sample/GroupQCOutputs.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 21 11:07:46 2018
+
+@author: jishuxu
+"""
```

Comment:
I think this auto-generated comment is unnecessary here since we have git to tell us who created/changed files and when :octocat: